packages/ui-toolkit/src/components/atoms/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -1,48 +1,44 @@
 import React from 'react';
 
 import './toggleSwitch.css';
+import '@groww-tech/mint-css/dist/index.css';
 
 
 const ToggleSwitch = (props: Props) => {
   const {
-    switchCircleColor, activeBackgroundColor, width,
-    inactiveBackgroundColor, onChange, height,
+    onChange, isDisabled,
     isActive, leftText, rightText, dataTestId
   } = props;
-
   // circleDiameter is the diameter of the circular slider which should be smaller than the size of the parent component so as to provide offset between the slider and it's parent
-  const circleDiameter = height - 4;
+  const circleDiameter = 20;
 
   const switchLabelStyle = {
-    background: isActive ? activeBackgroundColor : inactiveBackgroundColor,
-    width,
```

Comment:
we would need height, width

Reply:
But use case didn't show any height and width usage.